src/reporters/console/console-reporter.js

```diff
@@ -28,11 +28,6 @@ const tty = require('tty');
 type Row = Array<string>;
 type InquirerResponses<K, T> = {[key: K]: Array<T>};
 
-// fixes bold on windows
-if (process.platform === 'win32' && process.env.TERM && !/^xterm/i.test(process.env.TERM)) {
-  chalk.styles.bold.close += '\u001b[m';
-}
-
```

Comment:
We have to revert this commit I thought.

Reply:
Isn't it fixed on the new version?

Reply:
Nope?